operations/app/src/modules/database/main.tf

```diff
@@ -27,6 +27,18 @@ resource "azurerm_postgresql_server" "postgres_server" {
   }
 }
 
+data "azurerm_key_vault_key" "postgres_server_encryption_key" {
+  count = var.environment == "test" ? 1 : 0 // Only use Vormetric keys in test
```

Comment:
Is this just until we are fully ready to roll out to production?  Or did I miss something that we aren't doing this in prod?

Reply:
Yes, it's only in test for the moment. We need to have the Vormetric team push keys to prod once we validate the environment.

Reply:
I updated this to more easily change which environments are used by setting a variable that passes through.